src/effects/effectsmanager.cpp

```diff
@@ -1,497 +1,241 @@
 #include "effects/effectsmanager.h"
 
+#include <QDir>
 #include <QMetaType>
-#include <algorithm>
 
-#include "effects/effectchainmanager.h"
-#include "effects/effectsbackend.h"
+#include "control/controlpotmeter.h"
 #include "effects/effectslot.h"
-#include "engine/effects/engineeffect.h"
-#include "engine/effects/engineeffectchain.h"
-#include "engine/effects/engineeffectrack.h"
+#include "effects/effectsmessenger.h"
+#include "effects/presets/effectpresetmanager.h"
+#include "effects/presets/effectxmlelements.h"
+#include "effects/visibleeffectslist.h"
 #include "engine/effects/engineeffectsmanager.h"
 #include "moc_effectsmanager.cpp"
```

Comment:
```suggestion
```

`effectsmanager.h` doesn't contain QObject subclasses, so this causes an automoc warning:

    AutoMoc: /home/jan/Projects/mixxx/src/effects/effectsmanager.h:0: Note: No relevant classes found. No output generated.

Reply:
fixed 6c0676d